src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/incidents/IncidentsReconciliationService.kt

```diff
@@ -175,6 +175,7 @@ class IncidentsReconciliationService(
     if (nomis.staffParties.size != dps.staffInvolved.size) return "Staff parties mismatch"
     if (nomis.type != dps.nomisType) return "type mismatch"
     if (nomis.status.code != dps.nomisStatus) return "status mismatch"
+    if (nomis.reportedDateTime != dps.reportedAt) return "reported date mismatch"
```

Comment:
The reported date appears to be the most common data fix in Nomis for incidents, so have added to reconciliation to ensure any issues are picked up quickly